content/applications/sales/crm/pipeline/multi_sales_team.rst

```diff
@@ -2,24 +2,67 @@
 
-Use the *Sales Teams* feature to manage several sales teams, departments, or channels, each with
-their own unique sales processes.
+The *Multi Teams* feature in Odoo's *CRM* app allows salespeople to be assigned to multiple sales
+teams. Multiple teams can be created and managed in one dashboard, each with their own pipeline for
+leads and opportunities.
 
+Enable multi teams
+==================
 
-Create a new sales team
+To allow salespeople to be assigned to more than one sales team, the :guilabel:`Multi Teams` setting
+needs to be enabled. First, navigate to :menuselection:`CRM app --> Configuration --> Settings`.
+Under the :guilabel:`CRM` section, tick the checkbox labeled :guilabel:`Multi Teams`. Then, click
+:guilabel:`Save` at the top-left of the page.
+
+.. image:: multi_sales_team/multi-team-setting.png
+   :align: center
+   :alt: The Multi Teams setting in the CRM app.
+
+.. _crm/assignment-rules:
+
+Enable assignment rules
 =======================
 
-To create a new sales team, go to :menuselection:`CRM --> Configuration --> Sales Teams`, then
-click :guilabel:`Create`.
+To periodically :doc:`assign leads <../track_leads/lead_scoring>` based on rules, the
+:guilabel:`Rule-Based Assignment` feature needs to be enabled. Navigate to :menuselection:`CRM app
+--> Configuration --> Settings`. Under the :guilabel:`CRM` section, tick the checkbox labeled
+:guilabel:`Rule-Based Assignment`.
+
+This causes a new field, :guilabel:`Running`, to appear. Select an option from the drop-down menu in
+this field to manage how assignments are triggered.
+
+The :guilabel:`Rule-Based Assignment` feature can be set to run :guilabel:`Manually`, meaning an
+Odoo user must manually trigger the assignment, or :guilabel:`Repeatedly`, meaning Odoo
+automatically triggers the assignments.
+
+Selecting :guilabel:`Repeatedly` causes additional fields to appear. Use the :guilabel:`Repeat
+every` fields to select how frequently the rules should be triggered. The :guilabel:`Next Run` field
+automatically updates, though it can be manually updated, if needed.
+
+Once all desired configurations are complete, click :guilabel:`Save`.
+
+.. image:: multi_sales_team/rule-based-assignment.png
+   :align: center
+   :alt: The Rule-Based Assignment setting enabled in the CRM app.
 
-On the creation page, set an :guilabel:`Email Alias` to automatically generate a lead/opportunity
-for this sales team every time a message is sent to that unique email address. Choose whether to
-accept emails from :guilabel:`Everyone`, :guilabel:`Authenticated Partners`, or :guilabel:`Followers
-Only`.
+.. seealso::
+   For more information on assigning leads, see the :doc:`assign leads
+   <../track_leads/lead_scoring>` documentation.
 
-Set an :guilabel:`Invoicing Target` if this team has specific monthly revenue goals. Set a
-:guilabel:`Domain` to assign leads/opportunities to this sales team based on specific filters, such
-as country, language, or campaign.
+Create sales teams
```

Comment:
Similar feedback here...I would think the basics of creating and organizing sales teams in the CRM app would ideally be it's own doc, namely these two next sections starting at line 52.

As it is, this doc is coming across as a bit of a hybrid covering multiple topics and not necessarily how to manage multiple sales teams.